eslint-local-rules/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../tsconfig.utils.typecheck.json",
+  "compilerOptions": {
+    "module": "ESNext",
+    "moduleResolution": "Bundler",
+    "declaration": false
+  }
```

Comment:
I am curious to know why this is necessary?

Reply:
(Does not seem to be blocking, but question for you @SevInf)

Reply:
`module` and `modulResoultion` was necessary because `@typescript-eslint/utils` is ESM-only and does not resolve properly with our default settings. 
`declaration: false` - honestly, don't remember anymore. Will try removing it and see what happens.